OsmAnd/res/values-fr/phrases.xml

```diff
@@ -633,13 +633,13 @@
 	<string name="poi_transportation">Transport</string>
 	<string name="poi_man_made">Infrastructure</string>
 	<string name="poi_education">Éducation</string>
-	<string name="poi_administrative">Territoire</string>
+	<string name="poi_administrative">Administration</string>
 	<string name="poi_healthcare">Santé</string>
-	<string name="poi_office">Service</string>
+	<string name="poi_office">Tertiaire</string>
```

Comment:
It is the better one I've found.
"Bureau" is a literal translation, but it is really very strange for 
French (I assume you speak French, and so you'll agree).
"Service" is not so bad, but it is used for… "Service" POI !
As every POI in this "office" category belongs to tertiary sector, I 
think it is the best way to tag it.


Le 11/05/2015 12:16, Stéphane Brunner a écrit :
>
> In OsmAnd/res/values-fr/phrases.xml 
> <https://github.com/osmandapp/Osmand/pull/1150#discussion_r30027060>:
>
> >  	<string name="poi_healthcare">Santé</string>
> > -	<string name="poi_office">Service</string>
> > +	<string name="poi_office">Tertiaire</string>
>
> Is it relay a good translation ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/osmandapp/Osmand/pull/1150/files#r30027060>.
>